website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Custom Webpack Config"
+---
+
+By default Custom Applications include a webpack configuration to handle most of the common features like code transformation via Babel, loading CSS modules, loading images and SVGs, loading GraphQL documents, etc.
+However, it could be that your application requires some extra functionality like loading a particular file type, using a specific webpack plugin or include other source files in your repository.
+To extend the functionalities of webpack, you can define specific files in the root of your Custom Application project.
+
+# Usage
+The webpack configuration can be customized for both development and production.
+
+# Development
+For development mode, create a `webpack.config.dev.{js,cjs}` file and use the function `createWebpackConfigForDevelopment` to create and enhance the webpack configuration.
+The Custom Application CLI will then use the configuration exported from this file instead of the default one.
+
+```js title="webpack.config.dev.cjs"
+const { createWebpackConfigForDevelopment } = require("@commercetools-frontend/mc-scripts/webpack");
+
+// Create the default config
+const config = createWebpackConfigForDevelopment();
+
+// Customize the config
+config.module.rules = config.module.rules.concat({
+  test: /\.scss$/,
+  use: [
+    require.resolve('style-loader'),
+    require.resolve('css-loader'),
+    require.resolve('sass-loader'),
+  ],
+});
+
+// Export the config
+module.exports = config; 
+```
+
+# Production
+For production mode, create a `webpack.config.prod.{js,cjs}` file and use the function `createWebpackConfigForProduction` to create and enhance the webpack configuration.
+The Custom Application CLI will then use the configuration exported from this file instead of the default one.
+
+```js title="webpack.config.prod.cjs"
+const { createWebpackConfigForProduction } = require("@commercetools-frontend/mc-scripts/webpack");
+
+// Create the default config
+const config = createWebpackConfigForProduction();
+
+// Customize the config
+config.module.rules = config.module.rules.concat({
+  test: /\.scss$/,
+  use: [
+    require.resolve('style-loader'),
+    require.resolve('css-loader'),
+    require.resolve('sass-loader'),
+  ],
+});
+
+// Export the config
+module.exports = config; 
+```
+
+# Options
+
+## Include additional source folders
+You can list folders where Webpack should look for source files.
```

Comment:
nit: I find it a not very helpful the code snippet with only the `sourceFolder` prop, specially since we have a more meaningful example below.

Also, I would explain whether providing this prop will override the default source folders or they will be concatenated to them.

Reply:
Are we sure the provided source folders will overwrite the default ones?

Reply:
`options` [here](https://github.com/commercetools/merchant-center-application-kit/blob/dac14c2bf0800eed6ddd1977d988a31b23a9c47e/packages/mc-scripts/src/config/create-webpack-config-for-production.ts#L59) replaces the `defaultOptions` which contain the values for the default `sourceFolders`